pom.xml

```diff
@@ -846,26 +849,11 @@
         <artifactId>javax.annotation-api</artifactId>
         <version>${javax.annotation.api.version}</version>
       </dependency>
-      <dependency>
-        <groupId>org.ow2.asm</groupId>
-        <artifactId>asm</artifactId>
-        <version>${asm.version}</version>
```

Comment:
we use it directly and having this here in dptMngt will insure coherence between modules